Patches.py

```diff
@@ -1441,32 +1462,38 @@ def calculate_traded_flags(world):
     if any(hint_type in world.settings.misc_hints for hint_type in ('10_skulltulas', '20_skulltulas', '30_skulltulas', '40_skulltulas', '50_skulltulas')):
         rom.write_int16(0xEA185A, 0x44C8)
 
+    # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    # Does not appear to be used anymore.
+    # Some of these addresses conflict
+    # with scene files. Update before
+    # uncommenting.
+    # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
     # Patch freestanding items
-    if world.settings.shuffle_freestanding_items:
-        # Get freestanding item locations
-        actor_override_locations = [location for location in world.get_locations() if location.disabled == DisableType.ENABLED and location.type == 'ActorOverride']
-        rupeetower_locations = [location for location in world.get_locations() if location.disabled == DisableType.ENABLED and location.type == 'RupeeTower']
+    # if world.settings.shuffle_freestanding_items:
+    #     # Get freestanding item locations
+    #     actor_override_locations = [location for location in world.get_locations() if location.disabled == DisableType.ENABLED and location.type == 'ActorOverride']
+    #     rupeetower_locations = [location for location in world.get_locations() if location.disabled == DisableType.ENABLED and location.type == 'RupeeTower']
 
-        for location in actor_override_locations:
-            patch_actor_override(location, rom)
+    #     for location in actor_override_locations:
+    #         patch_actor_override(location, rom)
```

Comment:
This can be removed, it's no longer needed since these were refactored in potsanity 3.0.